supervisor/proxy/nslistener.go

```diff
@@ -213,19 +212,38 @@ func handleAddr(update netlink.AddrUpdate, callback func(supervisor.NetlinkUpdat
 
 // Route specific
 func handleRoute(update netlink.RouteUpdate, callback func(supervisor.NetlinkUpdate)) {
+	glog.V(3).Infof("[nslistener] route changed: %#v", update)
 	// Route type is not a bit mask for a couple of values, but a single
 	// unsigned int, that's why we use switch here not the "&" operator.
 	switch update.Type {
 	case syscall.RTM_NEWROUTE:
-		fmt.Printf("[Create a route]\t%+v\n", update)
+		glog.V(3).Infof("[Create a route]\t%+v\n", update)
 	case syscall.RTM_DELROUTE:
-		fmt.Printf("[Remove a route]\t%+v\n", update)
+		glog.V(3).Infof("[Remove a route]\t%+v\n", update)
 	case syscall.RTM_GETROUTE:
-		fmt.Printf("[Receive info of a route]\t%+v\n", update)
+		glog.V(3).Infof("[Receive info of a route]\t%+v\n", update)
 	}
 
 	netlinkUpdate := supervisor.NetlinkUpdate{}
 	netlinkUpdate.Route = update
 	netlinkUpdate.UpdateType = supervisor.UpdateTypeRoute
 	callback(netlinkUpdate)
+
+	// delete all the routes from outside
+	// The reason that we need to delete all the routes is
+	// that if we don't do this, we will get two active network interfaces,
+	// and ping the IP will get duplicate ICMP replies.
```

Comment:
@WeiZhang555 
Can you check the ipaddress of nic? is it same on two container? they should stay in the same netns.